plotly/tests/test_optional/test_opt_tracefactory.py

```diff
@@ -105,3 +105,54 @@ def test_simple_streamline(self):
         self.assertListEqual(strln['data'][0]['x'][0:100],
                              expected_strln_0_100['x'])
 
+
```

Comment:
@theengineear right now the format of colorscale is different from plotly JSON, as is the goal of using it. Colorscale in this case only replaces default colors that scipy assigns to cluster elements. There is no control over to which element the color will be assigned as it is usually is with Plotly's colorscale key-value pairs. 
Should I perhaps communicate this in a docstring/example?